libvast/src/system/index.cpp

```diff
@@ -664,6 +668,7 @@ void index_state::send_report() {
   auto query_counters = get_query_counters(pending_queries);
   auto msg = report{
     .data = {
+      {"catalog.lookup.false-positives", counters.partition_false_positives},
```

Comment:
Can you add the query metadata to this metric?

Reply:
I want to do that. But it requires us to issue it from the query completion function instead. I hope I can get to this on the sooner side.